taxi_service/settings.py

```diff
@@ -45,6 +45,9 @@
     "debug_toolbar",
     "crispy_forms",
     "taxi",
+    ...,
+   "crispy_bootstrap4",
+   "crispy_forms",
```

Comment:
There is a duplicate entry for 'crispy_forms' in the 'INSTALLED_APPS' list. Please remove one of them to avoid redundancy.